frontend/src/components/RFI/RFILanding.vue

```diff
@@ -852,152 +799,63 @@
-                      @click:minute="$refs.menuto[index].save(timeto)"
-                    />
-                  </v-menu>
+                    full-width
+                    max-width="200px"
+                    label="close"
+                  />
                 </v-col>
 
                 <v-col class="col-md-1 col-12">
-                  <v-menu
-                    ref="newmenufrom"
-                    v-model="obj.newmenufrom"
-                    v-model:return-value="newtimefrom"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field
-                        v-model="obj.newtimefrom"
-                        :disabled="isReadOnly"
-                        label="From"
-                        readonly
-                        v-bind="props"
-                      />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.newmenufrom"
-                      v-model="obj.newtimefrom"
-                      full-width
-                      @click:minute="$refs.newmenufrom[index].save(newtimefrom)"
-                    />
-                  </v-menu>
-                </v-col>
+                  <AppTimeInput
+                    v-model="obj.newtimefrom"
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="open"
+                  />
 
-                <v-col class="col-md-1 col-12">
-                  <v-menu
-                    ref="newmenuto"
-                    v-model="obj.newmenuto"
-                    v-model:return-value="newtimeto"
-                    :close-on-content-click="false"
-                    :nudge-right="40"
-                    transition="scale-transition"
-                    offset-y
-                    max-width="290px"
-                    min-width="290px"
-                  >
-                    <template #activator="{ props }">
-                      <v-text-field v-model="obj.newtimeto" :disabled="isReadOnly" label="To" readonly v-bind="props" />
-                    </template>
-                    <v-time-picker
-                      v-if="obj.newmenuto"
-                      v-model="obj.newtimeto"
-                      :min="obj.newtimefrom"
-                      full-width
-                      @click:minute="$refs.newmenuto[index].save(newtimeto)"
-                    />
-                  </v-menu>
+                  <AppTimeInput
+                    v-model="obj.newtimeto"
+                    :disabled="isReadOnly"
+                    full-width
+                    max-width="200px"
+                    label="close"
```

Comment:
I think the original label was 'To'. Do we have a requirement for changing it to 'close'?

Reply:
done